spec/helpers/time_helper_spec.rb

```diff
@@ -6,4 +6,14 @@
       expect(helper.timezone_select_options).to include(["Pacific Time (US & Canada)", "America/Los_Angeles"])
     end
   end
+
+  describe "#formatted_datetime" do
+    it "returns a string formatted to show the date and the time with zone" do
+      Time.use_zone("America/Los_Angeles") do
+        # in_time_zone coerces from -0800 -> PST
+        test_date = Time.zone.local(2004, 11, 24, 01, 04, 44).in_time_zone("America/Los_Angeles")
```

Comment:
I'm confused, what does the comment mean? Isn't -0800 the same as PST? It looks like it's converting from PST to PST. Was it returning a string with `-0800` before using `.in_time_zone`?
I don't know if a DateTime would behave any differently than Time, but it might be helpful to use a DateTime for specs since that is mostly the type it will be receiving from an ActiveRecord field.

Reply:
Yes, its converting the string of -0800 instead of PST and thats what the comment indicates. It's not an ideal way to create the date, but I didn't want to noodle on it too much for purposes getting a test going and trying to create a time value that mimicked what we're seeing.